cellxgene_schema_cli/cellxgene_schema/atac_seq.py

```diff
@@ -128,6 +131,15 @@
     "barcode": str,
     "read support": int,
 }
+schema = pa.schema(
```

Comment:
this make `column_types` obsolete above. Remove or replace `column_type` with `schema`.

Reply:
So, it looks like this also removes usage of `allowed_chromosomes` . Maybe we need to explicitly validate for this now?

Reply:
yep, take it out

Reply:
done!